### DISCUSSION TITLE
GSOC 2024 GUIDENCE

### DISCUSSION THREAD

**Discussion by QamarSayyad33** [polypheny/Polypheny-DB#474] [Q&A] [answered]
**Hello!**
I'm Qamar, and I'm interested in contributing to the **LDAP Query Interface** project for GSoC. I have expertise in Java and a solid understanding of LDAP queries. However, I'm new to contributing to open-source projects and would appreciate some guidance to get started.

Could you please help me by providing some direction on how to begin contributing to the project? Any resources, tips, or specific tasks I should focus on would be greatly appreciated.

**Comment by hennlo** [comment, marked as answer]
HI @QamarSayyad33 First of all welcome in our community: Feel free to join our [Discord Server](https://discord.gg/TC67WN5V)
to get all information around GSoC and get in touch with the rest of the community.
